frontend/types/api.ts

```diff
@@ -46,13 +46,3 @@ export interface PDSCreateSessionRequest {
   password: string;
 }
 export type PDSCreateSessionResponse = { [key: string]: any};
-export interface PDSGetRecordRequest {
-  repo: string;
-  collection: string;
-  rkey: string;
-}
-export interface PDSGetRecordResponse {
-  uri: string;
-  cid: string;
-  value: { [key: string]: any};
-}
```

Comment:
this automatically happened -- i'm guessing these got checked in in another PR but we're not actually using them yet? or something?

Reply:
see the fishtail stuff that got removed below too

Reply:
I think these were originally on the fishtail branch and split off later on, so probably fine